Makefile

```diff
@@ -86,6 +86,14 @@ conda-init:
 
 conda-update: conda-lock-all conda-init
 
+# Initialise CI environment
+mamba-init:
+	python3 requirements/check_conda_env.py
+	mamba config --env --add channels conda-forge --add channels eradiate
+	mamba update --file requirements/environment-$(PLATFORM).lock
+	python3 requirements/copy_envvars.py
+	pip install --editable . --no-deps
+
 .PHONY: conda-env conda-lock conda-lock-all conda-init conda-update
```

Comment:
You should add `mamba-init` to the list of phony targets to avoid any undesired behaviour.